targets/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -52,9 +52,11 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+/* Heap 1/4 of ram and stack 1/8. 0x1000 is reduced from both heap
+ * and stack to accomodate static data introduced by GCOV.
+ */
+__stack_size__ = 0x7000;
+__heap_size__ = 0x09000;
```

Comment:
I got the impression from Jimmy that this PR might not accepted because the changes do not comply with build profile (ex: debug) feature. So I started looking into an alternate approach with Fast models. However, this is something we can always revisit. Secondly, I have tested this with the largest module ```core/common``` at that time. It worked for all the tests. Getting stats for all the modules x tests is bit cumbersome. But we can visit that.